jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -50,6 +51,9 @@ def _add_to_table(app, data, comp_label, table_viewer_reference_name='table-view
     if data is None or len(data) == 0:
         return
 
+    # Convert any nans to strings for JSON serialization
+    data = [str(x) if pd.isna(x) else x for x in data]
```

Comment:
Is there a reason why we cannot blindly pass in str representation of everything for serialization?

```suggestion
    data = [str(x) for x in data]
```

Reply:
I think that depends on what is listening on the other side for the messages. If we change everything to strings, then we'll need to make sure to "recast" anything trying to use a message payload. Luckily for RA/Dec, most fields that would have "nans" would probably be string-compatible

Reply:
Whether who is listening I am not sure... I am just saying that if RA/Dec is the only thing tripping this check, pandas is probably unnecessary.

Reply:
Removed! See https://github.com/spacetelescope/jdaviz/pull/1958#issuecomment-1403934536